### ISSUE TITLE
TypeError when using `flip_ratio` metric

### ISSUE THREAD

**Issue by leonoverweel** [larq/larq#282]
### Describe the bug
When using the `flip_ratio` metric running imagenet2012 on Polyaxon, the model errors out with:

```
TypeError: An op outside of the function building code is being passed
a "Graph" tensor. It is possible to have Graph tensors
leak out of the function building context by including a
tf.init_scope in your function building code.
```

See full logs [here](https://polyaxon.plumerai.com/app/leon/phased-binarization/experiments/14188#logs) (internal link). This does not happen when running the same code locally on cifar10, and also does not happen when running the same code without the `flip_ratio` metric.

The problem could be related to [tensorflow 2.0.0-rc1 issue #32889](https://github.com/tensorflow/tensorflow/issues/32889) (opened in the last two weeks). 

### To Reproduce
Add `with larq.metrics.scope(["flip_ratio"]):` around your model building code.

### Expected behavior
The run does not error out.

### Environment

TensorFlow version: 2.0.0
Larq version: 0.7.3

**Comment by jneeven** [comment]
@leonoverweel  any idea if this is still an issue?

**Comment by leonoverweel** [comment]
When I do the end-to-end test for #402 (not a priority right now so probably next week) I'll do it on imagenet and see if this still happens. Thanks for flagging.

**Comment by lgeiger** [comment]
I think this was closed by #402, feel free to reopen if this issue comes up again.
